themes/docdock/layouts/partials/menu.html

```diff
@@ -1,5 +1,21 @@
 {{- $currentNode := . }}
 {{- $showvisitedlinks := .Site.Params.showVisitedLinks -}}
+
+{{/* Add back to workshops dashboard navigation if not on home page */}}
+{{- if not .IsHome -}}
+  <li class="dd-item back-to-dashboard">
+    <div class="menu-item">
+      <span>
+        Workshop Dashboard
```

Comment:
This will need to be a localized string

{{ i18n "____" }}

See examples in code